bika/lims/browser/worksheet/views/add_analyses.py

```diff
@@ -64,11 +64,11 @@ def __init__(self, context, request):
             'getClientOrderNumber': {
                 'title': _('Order'),
                 'index': 'getClientOrderNumber'},
-            'getRequestID': {
+            'getId': {
                 'title': _('Request ID'),
-                'attr': 'getRequestID',
-                'replace_url': 'getAnalysisRequestURL',
-                'index': 'getRequestID'},
+                'attr': 'getId',
+                'replace_url': 'getRequestURL',
```

Comment:
I see `getAnalysisRequetURL` is labelled as deprecated here https://github.com/senaite/bika.lims/blob/senaite-integration/bika/lims/content/abstractroutineanalysis.py#L338 in favour of `getRequestURL` https://github.com/senaite/bika.lims/blob/senaite-integration/bika/lims/content/abstractroutineanalysis.py#L174 , but in the catalog, the metadata column is still `getAnalysisRequestURL` https://github.com/senaite/bika.lims/blob/senaite-integration/bika/lims/catalog/analysis_catalog.py#L65 . As per the same reason discussed above (re UID vs ID), I think that indeed, is better to be coherent everywhere, so I'd use `getRequestURL` instead of `getAnalysisRequestURL` in the catalog

Reply:
Forget about this... I've seen now you already changed the index names in catalog.